.nuspec/Xamarin.Forms.nuspec

```diff
@@ -223,6 +224,7 @@
     <file src="..\Xamarin.Forms.Platform.UAP\bin\$Configuration$\StepperControl.xbf" target="lib\uap10.0\Xamarin.Forms.Platform.UAP" />
     <file src="..\Xamarin.Forms.Platform.UAP\bin\$Configuration$\CollectionView\ItemsViewStyles.xbf" target="lib\uap10.0\Xamarin.Forms.Platform.UAP\CollectionView" />
     <file src="..\Xamarin.Forms.Platform.UAP\bin\$Configuration$\FormsCheckBoxStyle.xbf" target="lib\uap10.0\Xamarin.Forms.Platform.UAP" />
+    <file src="..\Xamarin.Forms.Platform.UAP\bin\$Configuration$\Microsoft.UI.Xaml\DensityStyles\Compact.xbf" target="lib\uap10.0\Xamarin.Forms.Platform.UAP\Microsoft.UI.Xaml\DensityStyles\" />
```

Comment:
The Microsoft.UI.Xaml package should be including this for you. I'd worry this makes it get included twice + if you decide on using a newer version of WinUI, these might not align. When I was experimenting with using WinUI to support shell I didn't have to do this.

Reply:
@jevansaks Should this be necessary?

Reply:
No, especially with buildTransitive fix it shouldn't be necessary.

Reply:
Without it I get the following exception when I try to compile
```
Severity	Code	Description	Project	File	Line	Suppression State
Error		Payload file 'C:\Users\shane\.nuget\packages\xamarin.forms\4.3.0.2061\lib\uap10.0\Xamarin.Forms.Platform.UAP\Microsoft.UI.Xaml\DensityStyles\Compact.xbf' does not exist.	App133.UWP			
```

Which is perhaps caused by the addition of this?

https://github.com/xamarin/Xamarin.Forms/pull/7319#discussion-diff-319736060R8

Reply:
Here's a project with included nuget based on latest WINUI to demonstrate the exception above
[App133.zip](https://github.com/xamarin/Xamarin.Forms/files/3575999/App133.zip)

I also tested by removing
```
        <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls"/>
```

But same exception

Reply:
Hmm I see the problem. Apparently the resource gets referenced by `\lib\uap10.0\Xamarin.Forms.Platform.UAP.pri`, which is now missing.

Reply:
@jevansaks Perhaps this should only be done with project heads, not class libraries: 
https://github.com/microsoft/microsoft-ui-xaml/blob/f2b0459553152d69168aa6929c9ff0a7188ac8ea/build/NuSpecs/MUXControls-Nuget-FrameworkPackage.props#L134-L139

Reply:
@PureWeen Quick fix would be to explicitly remove that resource again in csproj, ie <Page Remove="**\*\DensityStyles\Compact.xaml" />.
But it's probably not a huge problem to have it there twice for now, as I'm guessing the ms-appx path used will be different.